src/pages/proposals/comment/author.js

```diff
@@ -30,19 +32,27 @@ export default class CommentAuthor extends React.Component {
         <span data-digix="CommentAuthor-QuarterPoints">
           {UserStats.quarterPoints}: {user.quarterPoint}
         </span>
+        {createdAt && (
+          <React.Fragment>
+            <span>&bull;</span>
+            <span data-digix="CommentAuthor-QuarterPoints">{getElapsedTime(createdAt)}</span>
```

Comment:
Please change the `data-digix` attribute to something like `CommentAuthor-ElapsedTime`. We use it for QA automation and this needs to be a unique and description value. Thanks!